.github/workflows/release.yml

```
@@ -1,42 +1,30 @@
-
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: pnpm release
+          publish: bun release
           commitMode: github-api
         env:
           GITHUB_TOKEN: ${{ secrets.JACOB_DEVELOPS_GH_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)